src/services/contentProgress.js

```diff
@@ -483,6 +490,15 @@ async function saveContentProgress(contentId, collection, progress, currentSecon
   const hierarchy = await getHierarchy(contentId, collection)
```

Comment:
Then after the response we could probably just skip the bubbling if the progress didn't change

```
if (progress == currentProgress) {
   return;
}
```